resources/troubleshooting-guides/push_notification_troubleshooting.md

```diff
@@ -0,0 +1,172 @@
+# Troubleshooting Campaign push notifications
+                         
+## To set up server key/APNS server certificate in the Campaign instance, complete the following steps:  
```

Comment:
Please make this regular body text and not a heading; as I'd explained during the call, this is an intro sentence for the 2 procedures.

Reply:
This statement belongs to first tab. Server key is generated for Android and APNS server certificate is generated for iOS in first step. Shouldn't it be heading for first tab?